foreshadow/preprocessor.py

```diff
@@ -1,715 +0,0 @@
-"""Core preprocessing transformer and helpers."""
```

Comment:
Instead of deleting the old one should we rename it for reference until we completely swap over? (of course we could mark it as nocover)

Reply:
sure.